### DISCUSSION TITLE
Support Playsessions/Statistics for Abudiobookshelf

### DISCUSSION THREAD

**Discussion by mutschler** [LeoKlaus/plappa#39] [Feature Requests]
It would be cool if plappa would report it's playsessions back to Audiobookshelf.
Currently only the play progress is synced (which is nice) but it would be cool to have playtime actually logged by audiobookshelf (statistics are amazing) ^^

Maybe even include the statistics in plappa directly?

**Comment by LeoKlaus** [comment]
Thank you again for the feedback!

Another user has already requested this in #15, so I'll close this one. I really like the idea of adding statistics in plappa, though!

**Reply by mutschler** [reply]
ah... i was confused about your mention about #13 guess you'll wanted to reference #15 there ;)

**Reply by LeoKlaus** [reply]
Oops, that’s correct.
